modules/images-configuration-insecure.adoc

```diff
@@ -50,16 +50,17 @@ spec:
     - registry.redhat.io
     - insecure.com <3>
     - reg4.io/myrepo/myapp:latest
+    - image-registry.openshift-image-registry.svc:5000
 status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ----
 <1> Contains configurations that determine how the container runtime should treat individual registries when accessing images for builds and pods. It does not contain configuration for the internal cluster registry.
-<2> Specify an insecure registry, or optionally a repository in that regisstry.
+<2> Specify an insecure registry. You can specify a repository in that registry.
 <3> Ensure that any insecure registries are included in the `allowedRegistries` list.
 +
 [NOTE]
 ====
-When the `allowedRegistries` parameter is defined, all registries, including the registry.redhat.io and quay.io registries, are blocked unless explicitly listed. If you use the parameter, to prevent pod failure, you must add `registry.redhat.io` and `quay.io` to the `allowedRegistries` list, as they are required by payload images within your environment. Do not add the `registry.redhat.io` and `quay.io` registries to the `blockedRegistries` list.
+When the `allowedRegistries` parameter is defined, all registries, including the registry.redhat.io and quay.io registries and the default internal image registry, are blocked unless explicitly listed. If you use the parameter, to prevent pod failure, add all registries including the `registry.redhat.io` and `quay.io` registries and the `internalRegistryHostname` to the `allowedRegistries` list, as they are required by payload images within your environment.
```

Comment:
It doesn't look like it's necessary, but does this note need the following line?

"For disconnected clusters, mirror registries should also be added."